index.js

```diff
@@ -753,28 +753,40 @@ function Device(atemIpAddress){
 	});
 
 	this.on('PrvI', function(d) {
-		const source = d.readUInt16BE(2),
+		const me = d.readUInt8(0);
+		const source = d.readUInt16BE(2)
 			inTransition = (d[4] & 1) === 1;
 
 		/**
 		 * When the selected preview bus changes this event will be fired.
 		 * @event Device#previewBus
-		 * @property {SourceID} source The new preview source
+		 * @property {number} me 0 for M/E 1, 1 for M/E 2
+		 * @property {number} source The new preview source
```

Comment:
Did the numbering of sources change? If yes: please elaborate on the new numbering structure, if not: please use the `SourceID` instead of `number`.

Reply:
My apologies - no reason for me to have changed this!